lua/oil/keymap_util.lua

```diff
@@ -92,10 +92,18 @@ M.show_help = function(keymaps)
     end
   end
 
+  local sorted_indices = {}
+  for i = 1, #col_desc do
+    sorted_indices[i] = i
+  end
+  table.sort(sorted_indices, function(i, j)
+    return col_desc[i] < col_desc[j]
+  end)
+
```

Comment:
If we're going to have to sort these, I think it would be cleaner to merge the `col_left` and `col_desc` tables into a single list of `{<keymap>, <desc>}` tuples and sort that directly

Reply:
Fixed it as you suggested.